go/vt/mysqlctl/mysqld.go

```diff
@@ -688,7 +687,12 @@ func deleteTopDir(dir string) (removalErr error) {
 // executeMysqlCommands executes some SQL commands,
 // using the mysql command line tool.
 func (mysqld *Mysqld) executeMysqlCommands(connParams *sqldb.ConnParams, sql string) error {
-	return mysqld.executeMysqlScript(connParams, strings.NewReader(sql))
+	params, err := dbconfigs.WithCredentials(connParams)
```

Comment:
executeMysqlCommands is only used by the schema side, with dba credentials. So I moved it there, and renamed it. Much clearer now.